jkanban.js

```diff
@@ -54,8 +54,8 @@ var dragula = require('dragula');
       dragBoard: function (el, source) {},
       dragendBoard: function (el) {},
       dropBoard: function (el, target, source, sibling) {},
-      click: function (el) {},
-      context: function (el, e) {},
+      click: null,
+      context: null,
```

Comment:
Thinking about that, I could suggest you to create a property called ownedHandlers or so, in order to control what event jKanban will handle ('click', 'context').